core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2660,3 +2660,16 @@ secret_configs:
   - cluster: vsphere
     name: github-client-secret
     namespace: openshift-config
+- from:
```

Comment:
That is a subset of the push secret we have everywhere or not? Can we just reuse that instead? If not, we should rename this to indicate what it does, `dptp-internal` is sth I would expect to have admin perms everywhere

Reply:
> That is a subset of the push secret we have everywhere or not?
No, missing pull permissions. See the PR description above.

I will rename the secret.